react-graphql-apollo-tutorial.md

```diff
@@ -2492,7 +2484,7 @@ const Input = ({ children, color = 'black', ...props }) => (
 export default Input;
 {{< /highlight >}}
 
-Even though the search field works in the Navigation component, it doesn't help the rest of the application. It only updates the state in the Navigation component when a search request is submitted. However, the value of the search request is needed in the Organization component, one component above the Navigation component, as GraphQL variable for the query. That's why the local state needs to be lifted up from the Navigation component to the App component. The Navigation component becomes a stateless functional component again.
+While the search field works in the Navigation component, it doesn't help the rest of the application. It only updates the state in the Navigation component when a search request is submitted. However, the value of the search request is needed in the Organization component as a GraphQL variable for the query, so the local state needs to be imported from the Navigation component to the App component. The Navigation component becomes a stateless functional component again.
```

Comment:
"the local state needs to be imported" to "the local state needs to be lifted up "

- lifting state up and down is a thing in React
- import can be mistaken with the import/export statements

Reply:
Understood.  Learning React as I go along myself. :-)